src/problem-matcher.js

```diff
@@ -0,0 +1,42 @@
+import { info } from '@actions/core';
+import { writeFileSync } from 'fs';
+
+const json = {
+    problemMatcher: [
+        {
+            owner: 'packsquash-error',
+            severity: 'error',
+            pattern: [
+                {
+                    regexp: '^[!❌] (?!.*Invalid stick parity bit)(.+)$',
+                    message: 1
+                }
+            ]
+        },
+        {
+            owner: 'packsquash-warning',
+            severity: 'warning',
+            pattern: [
+                {
+                    regexp: '^[*⚡] (.+)$',
+                    message: 1
+                }
+            ]
+        }
+    ]
+};
+
+/**
+ * @param {string} path
+ */
+export function addProblemMatcher(path) {
+    writeFileSync(path, JSON.stringify(json));
+    info(`##[add-matcher]${path}`);
+}
```

Comment:
Make everything `async`! :smile: 

```suggestion
export async function addProblemMatcher(path) {
    await writeFile(path, JSON.stringify(json));
    info(`##[add-matcher]${path}`);
}
```

Reply:
Fix by b6b30e7f70e736cf727e199c89fe64280ce26466